src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -346,6 +347,113 @@ static bool _appendShape(SvgNode* node, Shape* shape, float vx, float vy, float
     return true;
 }
 
+enum class imageMimeTypeEncoding
+{
+    base64 = 0x1,
+    utf8 = 0x2
+};
+constexpr imageMimeTypeEncoding operator|(imageMimeTypeEncoding a, imageMimeTypeEncoding b) {
+    return static_cast<imageMimeTypeEncoding>(static_cast<int>(a) | static_cast<int>(b));
+}
+constexpr bool operator&(imageMimeTypeEncoding a, imageMimeTypeEncoding b) {
+    return (static_cast<int>(a) & static_cast<int>(b));
+}
+
+static constexpr struct
+{
+    const char* name;
+    int sz;
+    imageMimeTypeEncoding encoding;
+} imageMimeTypes[] = {
+#ifdef THORVG_JPG_LOADER_SUPPORT
+    {"jpeg", sizeof("jpeg"), imageMimeTypeEncoding::base64},
+#endif
+#ifdef THORVG_PNG_LOADER_SUPPORT
```

Comment:
I think it's bad that SVG has PNG loader dependency. Thorvg provides Picture which abstracts them. So just use picture  is enough to go.

Reply:
removed `#ifdef`